macros/measures.sql

```diff
@@ -5,7 +5,7 @@
 {%- endmacro -%}
 
 {%- macro default__measure_row_count(column_name, data_type) -%}
-cast(count(*) as {{ dbt.type_int() }})
+cast(count(*) as {{ dbt.type_numeric() }})
```

Comment:
This was changed to bigint in https://github.com/data-mie/dbt-profiler/pull/95. Does oracle support bigints?

Reply:
Oracle does not support bigint. I will create a separate macro for Oracle.